code_generator_funcadl_xAOD/pyproject.toml

```diff
@@ -8,7 +8,7 @@ packages = [{ include = "code_generator_funcadl_xaod" }]
 
 [tool.poetry.dependencies]
 python = "~3.10"
-func-adl-xAOD = "^2.2.0"
+func-adl-xAOD = "~2.2"
```

Comment:
Is this actually better? The original will allow any 2.x, the new one will only show 2.2.x

Reply:
Ah, no, in this case the caret also requires 2.2.x, so they should be identical. In any case, do you want to keep 2.2.x or allow 2.x ?